core/src/templates/components/alert/alert.twig

```diff
@@ -19,13 +19,13 @@
  * - alert_body: Text details of the alert message.
  */
 #}
-<div {{ attributes }} class="su-alert {{ modifier_class }}" >
-  <div class="su-alert__body">
+<article {{ attributes }} class="su-alert {{ modifier_class }}">
```

Comment:
After the discussion in #474 I think `<div>` is the appropriate element after all.